public/js/admin_action.js

```diff
@@ -0,0 +1,114 @@
+function actionInfoToForm(ref_id, status, mails){
+  $('#ref_id_input_id').val(ref_id);
+  $('#status_input_id').val(status);
+  $('#table_mails_action').prepend(createTable(mails[ref_id]));
+}
+
+
+ function renderSubmitButtonAdminActionForm(){
+ 	ck1 = document.getElementById('checkboxAdminActionForm1') ;
```

Comment:
Use jquery methods instead of raw javascript (for the whole file, not only this line)